src/main/frontend/components/page.cljs

```diff
@@ -1055,7 +1055,7 @@
                                                   all? (= 1 @*indeterminate)]
                                               (doseq [{:block/keys [idx]} @*results]
                                                 (swap! *checks assoc idx (or indeterminate? (not all?))))))
-                           :indeterminate (= -1 @*indeterminate)})]
+                           :indeterminate (when (= -1 @*indeterminate) "indeterminate")})]
```

Comment:
Wait, is there any reference of setting the value of `.-indeterminate` as string `"indeterminate"`?
Seems the only options here are `"true"` or `"false"`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#indeterminate_state_checkboxes

Reply:
This is a html5 to xhtml conversion. The warning message is wrong.

In html5, you write `<input checked>`.
In xhtml, you write `<input checked="checked">`, not `<input checked="true">`.